sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -273,7 +277,8 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentFromUr
         /// <para>See <a href="https://aka.ms/formrecognizer/receiptfields"/> for a list of available fields on a receipt.</para>
         /// </summary>
         /// <param name="receipt">The stream containing the one or more receipts to recognize values from.</param>
-        /// <param name="recognizeReceiptsOptions">A set of options available for configuring the recognize request.</param>
+        /// <param name="recognizeReceiptsOptions">A set of options available for configuring the recognize request. For example, specify the content type of the
+        /// form, the locale of the form, or whether or not to include form elements.</param>
```

Comment:
since we're listing all the options here, it might be worth mentioning that the includeFieldElements option also controls whether you'll get back all the lines on the page. Or maybe on the parameter itself, actually.

Reply:
So the [parameter](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/formrecognizer/Azure.AI.FormRecognizer/src/RecognizeReceiptsOptions.cs#L22) itself says:
>Whether or not to include form elements such as lines and words in addition to form fields.

Do you think I need to be more specific on it?

Reply:
I saw Python's and I like it more :P I will copy that :)
>Whether or not to include all lines per page and field elements such as lines, words,
            and selection marks for each form field.

Reply:
Hmm not sure I would connect that with the list of `lines` being populated on `FormPage`. I recently changed my doc for this to "Whether or not to include all lines per page and field elements such as lines, words, and selection marks for each form field.".

Reply:
😄 I didn't see your previous post until after I commented

Reply:
No worries :)